pmm/pmm2-ovf-staging-start.groovy

```diff
@@ -32,78 +61,169 @@ pipeline {
+                        sleep 60
+                        curl -s --user admin:admin http://${IP}/v1/Settings/Change --data '{"ssh_key": "'"\${OVF_PUBLIC_KEY}"'"}'
+                    """
+                }
+            }
+        }
+        stage('Enable Testing Repo') {
+            when {
+                expression { env.ENABLE_TESTING_REPO == "yes" && env.ENABLE_EXPERIMENTAL_REPO == "no" }
+            }
+            steps {
+                node(env.VM_NAME){
+                    enableRepo('testing', env.IP)
+                }
+            }
+        }
+        stage('Enable Experimental Repo') {
+            when {
+                expression { env.ENABLE_EXPERIMENTAL_REPO == "yes" && env.ENABLE_TESTING_REPO == "no" }
+            }
+            steps {
+                node(env.VM_NAME){
+                    enableRepo('experimental', env.IP)
+                }
+            }
+        }
+        stage('Enable Release Repo') {
+            when {
+                expression { env.ENABLE_EXPERIMENTAL_REPO == "no" && env.ENABLE_TESTING_REPO == "no" }
+            }
+            steps {
+                node(env.VM_NAME) {
+                    withCredentials([sshUserPrivateKey(credentialsId: 'OVF_VM_TESTQA', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
+                        sh '''
+                            ssh -i "${KEY_PATH}" -p 3022 -o ConnectTimeout=1 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null admin@${IP} '
+                                sudo yum update -y percona-release || true
+                                sudo yum clean all
+                            '
+                        '''
+                    }
+                }
+            }
+        }
+        stage('Setup QA Repo on OVF VM') {
+            steps {
+                node(env.VM_NAME) {
+                    withCredentials([sshUserPrivateKey(credentialsId: 'OVF_VM_TESTQA', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
+                        sh """
+                            ssh -i "${KEY_PATH}" -p 3022 -o ConnectTimeout=1 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null admin@${IP} '
+                                export PMM_QA_GIT_BRANCH=${PMM_QA_GIT_BRANCH}
+                                export PMM_QA_GIT_COMMIT_HASH=${PMM_QA_GIT_COMMIT_HASH}
+                                sudo yum install -y wget git
+                                sudo mkdir -p /srv/pmm-qa || :
+                                pushd /srv/pmm-qa
+                                    sudo git clone --single-branch --branch ${PMM_QA_GIT_BRANCH} https://github.com/percona/pmm-qa.git .
+                                    sudo git checkout ${PMM_QA_GIT_COMMIT_HASH}
+                                    sudo wget https://raw.githubusercontent.com/Percona-QA/percona-qa/master/get_download_link.sh
+                                    sudo chmod 755 get_download_link.sh
```

Comment:
maybe chmod +x ?

Reply:
Could be, `755` does more and is more explicit